BackofficeBundle/Manager/NodeManager.php

```diff
@@ -92,6 +97,20 @@ public function createNewLanguageNode(NodeInterface $node, $language)
     }
 
     /**
+     * @param NodeInterface $node
+     *
+     * @return StatusInterface
+     */
+    protected function getInitialStatus(NodeInterface $node)
+    {
+        if ($node->getNodeType() == NodeInterface::TYPE_GENERAL) {
```

Comment:
Yes, but they can be TYPE_GENERAL nodes which are not transverse node.

The point of my remark was not about that but about the different test you where doing in the nodeTransformer and at this line